frontend/src/metabase/query_builder/components/template_tags/SnippetSidebar/SnippetSidebar.jsx

```diff
@@ -58,7 +55,7 @@ class SnippetSidebarInner extends React.Component {
       className={S.SidebarFooter}
       onClick={() => this.setState({ showArchived: true })}
     >
-      <Icon className={S.SidebarIcon} name="view_archive" size={ICON_SIZE} />
+      <Icon className={S.SidebarIcon} name="view_archive" />
```

Comment:
"Archived snippets" header is styled differently than "Snippets" header: [snippets](https://github.com/user-attachments/assets/ae753cd2-7765-47d1-bf69-781deeef08e0) vs [archived snippets](https://github.com/user-attachments/assets/6876655a-8eba-4280-9043-e8e97e76b26b) (font, padding)